consensus/istanbul/backend/handler_test.go

```diff
@@ -84,8 +85,15 @@ func TestHandleNewBlockMessage_whenTypical(t *testing.T) {
 	arbitraryBlock, arbitraryP2PMessage := buildArbitraryP2PNewBlockMessage(t, false)
 	postAndWait(backend, arbitraryBlock, t)
 
-	handled, err := backend.HandleMsg(arbitraryAddress, arbitraryP2PMessage)
-
+	var handled bool
+	var err error
+	for i := 0; i < 5; i++ { // make 5 tries if a little wait
+		handled, err = backend.HandleMsg(arbitraryAddress, arbitraryP2PMessage)
```

Comment:
could we create a utility function in the same file to avoid the duplication across 3 tests?